voluncheer/opportunityboard/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from opportunityboard.views.opportunityboard import opportunityboard, select
+from opportunityboard.views.postanopportunity import post_an_opportunity
+
+
+urlpatterns = [
+    path("", opportunityboard, name="opportunityboard"),
+    # Opportunityboard
+    path("select", select, name="select"),
+    # Post an Opportunity
+    path("post_an_opportunity", post_an_opportunity, name="post_an_opportunity"),
```

Comment:
So this is certainly the thing I understand least about django. Is this path (post_an_opportunity) the URL path the user will see/enter in the browser? If so, we should rethink it but if it is just the path the app directs to internally it is fine.

Reply:
Yes, it is an url path. For now we can add guard clauses to our views in order to defend unauthorized users. But we may need to find a better way

Reply:
I changed the url from "post_an_opportunity" to "post." Full path is "opportunityboard/post."